src/redux/home/homeSlice.js

```diff
@@ -0,0 +1,512 @@
+    const pollution7Resp = await axios.get(`https://api.openweathermap.org/data/2.5/air_pollution?lat=${coords[6].lat}&lon=${coords[6].lon}&appid=c435ab8224ec4a9e751a3469cb551dde`);
+    const weather7Resp = await axios.get(`https://api.openweathermap.org/data/2.5/weather?lat=${coords[6].lat}&lon=${coords[6].lon}&appid=c435ab8224ec4a9e751a3469cb551dde`);
+    if (city7Resp && pollution7Resp && weather7Resp) {
+      const city7 = {
+        city: city7Resp.data[0].name,
+        aqi: pollution7Resp.data.list[0].main.aqi,
+        pollution: pollution7Resp.data.list[0].components,
+        weather: {
+          temp: weather7Resp.data.main.temp,
+          hu: weather7Resp.data.main.humidity,
+          ws: weather7Resp.data.wind.speed,
+          icon: weather7Resp.data.weather[0].icon,
+        },
+      };
+      otherCities.push(city7);
+    }
+
+    // city 8
+    const city8Resp = await axios.get(`https://api.openweathermap.org/geo/1.0/reverse?lat=${coords[7].lat}&lon=${coords[7].lon}&limit=1&appid=c435ab8224ec4a9e751a3469cb551dde`);
+    const pollution8Resp = await axios.get(`https://api.openweathermap.org/data/2.5/air_pollution?lat=${coords[7].lat}&lon=${coords[7].lon}&appid=c435ab8224ec4a9e751a3469cb551dde`);
+    const weather8Resp = await axios.get(`https://api.openweathermap.org/data/2.5/weather?lat=${coords[7].lat}&lon=${coords[7].lon}&appid=c435ab8224ec4a9e751a3469cb551dde`);
+    if (city8Resp && pollution8Resp && weather8Resp) {
+      const city8 = {
+        city: city8Resp.data[0].name,
+        aqi: pollution8Resp.data.list[0].main.aqi,
+        pollution: pollution8Resp.data.list[0].components,
+        weather: {
+          temp: weather8Resp.data.main.temp,
+          hu: weather8Resp.data.main.humidity,
+          ws: weather8Resp.data.wind.speed,
+          icon: weather8Resp.data.weather[0].icon,
+        },
+      };
+      otherCities.push(city8);
+    }
+    // Fetch Nearest City data End
+
+    switch (airPollutionIndex) {
+      case 1:
+        pollutionLevel = 'Good';
+        break;
+      case 2:
+        pollutionLevel = 'Fair';
+        break;
+      case 3:
+        pollutionLevel = 'Moderate';
+        break;
+      case 4:
+        pollutionLevel = 'Poor';
+        break;
+      case 5:
+        pollutionLevel = 'Very Poor';
+        break;
+      default:
+        pollutionLevel = '';
+    }
+    return (
+      {
+        city: {
+          country,
+          state,
+          city: cityName,
+          aqi: airPollutionIndex,
+          pollutionLevel,
+          pollution: pollutionResp.data.list[0].components,
+          weather: {
+            temp: weatherResp.data.main.temp,
+            hu: weatherResp.data.main.humidity,
+            ws: weatherResp.data.wind.speed,
+            icon: weatherResp.data.weather[0].icon,
+          },
+        },
+        otherCities,
+      }
+    );
+  } catch (error) {
+    return rejectWithValue('Your Location is not supported !!');
```

Comment:
**[OPTIONAL]**

- Hey, at the start of your page, it throws the error for some specific country, e.g Nigeria, the app would throw that error, The app should be accessible to any country of the world and as such, please, kindly improve on this to make the app accessible to any country to improve the user experience of your app.